dces-drc-integration/build.gradle

```diff
@@ -121,6 +125,7 @@ dependencies {
 	testImplementation 'io.projectreactor:reactor-test'
 	testImplementation "org.wiremock:wiremock-standalone:$versions.wiremockVersion"
 	testImplementation "com.squareup.okhttp3:mockwebserver:$versions.mockwebserverVersion"
+	testImplementation "org.flywaydb:flyway-core:6.4.3"
```

Comment:
is there a reason why the testImplementaion version is different to  implementation "org.flywaydb:flyway-core:$versions.flywayVersion" which is 10.20.0 

Does it matter?

Reply:
Nope, that's my mistake. Updated to latest as well.